torch/csrc/jit/passes/frozen_graph_optimizations.cpp

```diff
@@ -21,6 +25,8 @@ void OptimizeFrozenGraph(
       FoldFrozenConvMulOrDiv(graph);
     }
   }
+
+  FuseFrozenConvRelu(graph);
```

Comment:
I think we might want to integrate this into the main api only after a little more testing on benchmark suite and such.

Reply:
Let me revert this one.